### PR TITLE
Update fancyzones.md

### PR DESCRIPTION
**Pull request by crutkas**

### CODE REVIEW

**Comment by PRMerger14** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by noraa-junker — APPROVED** [review]
LGTM

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

File: hub/powertoys/fancyzones.md [hunk 1]
```diff
@@ -54,21 +75,23 @@ The **Distance to highlight adjacent zones** enables you to set a custom value f
...
 The zones editor also supports creating and saving custom layouts. Select the <kbd>+ Create new layout</kbd> button at the bottom-right.
   
-There are two ways to create custom zone layouts: **Grid** layout and **Canvas** layout. These can also be thought of as subtractive and additive models.
+There are two ways to create custom zone layouts: **Grid** layout and **Canvas** layout. 
 
-The subtractive **Grid** model starts with a three column grid and allows zones to be created by splitting and merging zones, resizing the gutter between zones as desired.
+The **Grid** model starts with a three column grid and allows zones to be created by splitting and merging zones, resizing the gutter between zones as desired. This is a relative layout and will resize with different screen sizes.
 
-To merge two zones, select and hold the left mouse button and drag the mouse until a second zone is selected, then release the button and a popup menu will show up.
+- To divide a zone, just click your mouse.  To rotate the
```
Comment (Jay-o-Way) [inline hub/powertoys/fancyzones.md]: This sentence is cut off?

File: hub/powertoys/fancyzones.md [hunk 2]
```diff
@@ -54,21 +75,23 @@ The **Distance to highlight adjacent zones** enables you to set a custom value f
...
 
-To merge two zones, select and hold the left mouse button and drag the mouse until a second zone is selected, then release the button and a popup menu will show up.
+- To divide a zone, just click your mouse.  To rotate the 
+- To move a divider, click on the thumb and drag.
+- To merge zones, select and hold the left mouse button and drag the mouse until multiple zones are selected, then release the button and a popup menu will show up.  Click "Merge" and they will become one big zone.
 
 ![FancyZones Table Editor Mode](../images/pt-fancyzones-grideditor.png)
 
-The additive **Canvas** model starts with a blank layout and supports adding zones that can be dragged and resized similar to windows.
+The additive **Canvas** model starts with a blank layout and supports adding zones that can be dragged and resized similar to windows.  Canvas layouts are directly tied to DPI and resolution of the monitor.
```
Comment (Jay-o-Way) [inline hub/powertoys/fancyzones.md:88]: Removing the words "additive"?

**Comment by Jay-o-Way** [comment]
Saw in line 111: the first `(` is supposed to be immediately before "with the Shift..."

File: hub/powertoys/fancyzones.md [hunk 3]
```diff
@@ -13,11 +13,32 @@ FancyZones is a window manager utility for arranging and snapping windows into e
...
+## Getting started snapping windows
 
-### Enable
+If two zones are adjacent, a window can be snapped to the sum of their area (rounded to the minimum rectangle that contains both). When the mouse cursor is near the common edge of two zones, both zones are activated simultaneously, allowing you to drop the window into both zones.
+
+It's also possible to snap to any number of zones: first drag the window until one zone is activated, then press and hold the <kbd>Control</kbd> key while dragging the window to select multiple zones.
+
+![Two Zones Activation screenshot](../images/pt-fancyzones-twozones.png)
+
+To snap a window to multiple zone using only the keyboard, first turn on the setting **Override Windows Snap** and select **Move windows based on: Relative position** as shown below. After snapping a window to one zone, use <kbd>Win</kbd>+<kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>[arrows]</kbd> to expand the window to multiple zones.
```
Comment (Jay-o-Way) [inline hub/powertoys/fancyzones.md:24]: ...multiple zone`s`... (must have missed that earlier)

File: hub/powertoys/fancyzones.md [hunk 4]
```diff
@@ -28,9 +49,9 @@ Launch the zones editor using the button in the PowerToys Settings menu or by pr
 
 ![FancyZones Settings UI](../images/pt-fancyzones-settings.png) 
 
-### Elevated permission admin apps
+### Snapping apps with elevated permission
 
-If you have applications that are elevated, run PowerToys in administrator mode. Read [PowerToys and running as administrator](administrator.md) for more information.
+If you have applications that are elevated and want them snapped with FancyZones, such as Windows Terminal or Task Manager, please run PowerToys in administrator mode. Read [PowerToys and running as administrator](administrator.md) for more information.
```
Comment (Jay-o-Way) [inline hub/powertoys/fancyzones.md]:
```suggestion
If you have applications that are elevated (such as Windows Terminal or Task Manager) and want them snapped with FancyZones, please run PowerToys in administrator mode. Read [PowerToys and running as administrator](administrator.md) for more information.
```

File: hub/powertoys/fancyzones.md [hunk 5]
```diff
@@ -54,21 +75,23 @@ The **Distance to highlight adjacent zones** enables you to set a custom value f
 
 The zones editor also supports creating and saving custom layouts. Select the <kbd>+ Create new layout</kbd> button at the bottom-right.
   
-There are two ways to create custom zone layouts: **Grid** layout and **Canvas** layout. These can also be thought of as subtractive and additive models.
+There are two ways to create custom zone layouts: **Grid** layout and **Canvas** layout. 
 
-The subtractive **Grid** model starts with a three column grid and allows zones to be created by splitting and merging zones, resizing the gutter between zones as desired.
+The **Grid** model starts with a three column grid and allows zones to be created by splitting and merging zones, resizing the gutter between zones as desired. This is a relative layout and will resize with different screen sizes.
```
Comment (Jay-o-Way) [inline hub/powertoys/fancyzones.md:80]: sizes/resolutions/scaling..?

**Comment by Jay-o-Way** [comment]
The list of settings shows "Zone highlight/inactive/border color" with default values. But latest version uses Windows theme colors. Simply removing the "default" text would be good.

File: hub/powertoys/fancyzones.md (hunk 2 above)
Comment (Jay-o-Way) [inline hub/powertoys/fancyzones.md]:
```suggestion
The **Canvas** model starts with a blank layout and supports adding zones that can be dragged and resized similar to windows. Canvas layouts are directly tied to DPI and resolution of the monitor.
```

File: hub/powertoys/fancyzones.md (hunk 3 above)
Comment (Jay-o-Way) [inline hub/powertoys/fancyzones.md]:
```suggestion
To snap a window to multiple zones using only the keyboard, first turn on the setting **Override Windows Snap** and select **Move windows based on: Relative position** as shown below. After snapping a window to one zone, use <kbd>Win</kbd>+<kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>[arrows]</kbd> to expand the window to multiple zones.
```

**Comment by PRMerger9** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@crutkas : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger19** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@mattwojo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by mattwojo — APPROVED** [review]
Made some minor updates and resolved the default color issue that Jay raised

**Pull request merged**